### ISSUE TITLE
Integrate with an admin panel

### ISSUE THREAD

**Issue by bilogic** [ondras/wwwsqldesigner#316]
![image](https://user-images.githubusercontent.com/946010/189514377-ac31008a-001e-43fd-927f-55ea6454e2e8.png)

1. When I drag on the canvas, the dragged area is away from where my cursor is
2. Was this designed to integrate "easily" with other HTML/CSS?
3. Has anyone else manage to do so?

**Comment by bilogic** [comment]
I took a closer look, it seems that sqldesigner accounts for only width and height, this implicitly assumes the canvas always starts at 0, 0.

**Comment by ondras** [comment]
> 2\. Was this designed to integrate "easily" with other HTML/CSS?

Nope. This was meant as a standalone app. But there are certainly ways to introduce/increase interoperability.
